custom-plugin-rules/code-sample-checks/check-sdk-coverage.js

```diff
@@ -0,0 +1,33 @@
+module.exports = CheckSDKCoverage;
+
+// If you have different SDK languages edit this array
+const sdkLanguages = ['bash', 'javascript', 'python', 'ruby', 'java', 'kotlin'];
+
+function CheckSDKCoverage () {
+    return {
+        XCodeSampleList: {
+          enter(codeSampleList, ctx) {
+            // Make sure the list contains at least one bash sample
+            const hasBashSample = codeSampleList.some((codeSample) => {
+              return codeSample.lang === 'bash';
```

Comment:
This is a bit weird -- is this a curl example? usually people would describe it as curl and it works with a variety of shells?

Reply:
This is already merged, but yes. I think this was to do with the syntax highlighter on the render library we're using.